src/runtime/mod.rs

```diff
@@ -152,9 +150,6 @@ pub struct Runtime {
 
 #[derive(Debug)]
 struct Inner {
-    /// Reactor running on a background thread.
-    reactor: Background,
-
     /// Task execution pool.
     pool: threadpool::ThreadPool,
 }
```

Comment:
Should I remove `Inner` since now it only contains the `ThreadPool`? Doesn't make much sense to go through `.inner.pool` rather than just `.pool`.